app/assets/stylesheets/components/_status.scss

```diff
@@ -6,7 +6,12 @@
 }
 
 .status__icon--exclamation {
-  fill: $color-yellow-dark;
+  background-color: $color-yellow-light;
+  border-radius: 100%;
```

Comment:
Yeah that was my understanding too, but I was getting an oblong on mobile for some reason. Might have been something else causing that mind you. My brain is full of the cold :)